pom.xml

```diff
@@ -413,7 +413,7 @@
       <dependency>
         <groupId>org.mongodb</groupId>
         <artifactId>mongo-java-driver</artifactId>
-        <version>3.12.10</version>
+        <version>3.12.9</version>
```

Comment:
it seems like this driver version might be compatible with mongodb version 3.7 (https://github.com/cBioPortal/cbioportal-docker-compose/blob/a8759d8751ecc6ba172a67b06e1b40cd4ac5b8bf/docker-compose.yml#L49):  https://docs.mongodb.com/drivers/java/sync/current/compatibility/. For some reason mongodb version 3.7 is not specifically listed (maybe it's not LTS) or something

Reply:
anyway it's just a patch version change so i guess it should be fine